notebooks/220-yolov5-accuracy-check-and-quantization/220-yolov5-accuracy-check-and-quantization.ipynb

```diff
@@ -46,11 +46,13 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.    optimized_save_dir = Path(Path.cwd()).joinpath(config["model_int8"]["save_path"])
</pre><p>probably </p><p>Path(config["model_int8"]["save_path"])) should be enough</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thank you for your suggestion. I agree that using Path(config["model_int8"]["save_path"]) directly would simplify the code.